### PR TITLE
Update lang-ro.lua

### PR DESCRIPTION
**Pull request by Generalyuri342**
Rip I done it now

### CODE REVIEW

File: Localization/lang-ro.lua [hunk 1]
```diff
@@ -94,10 +94,136 @@ module.GetDate = function(day,month,year)
...
+    ETHNICITY_QUECHUA = "Quechua";
+    ETHNICITY_ROMANIAN = "român";
+    ETHNICITY_RUSSIAN = "Rus";
+    ETHNICITY_SAMOAN = "Samoan";
+    ETHNICITY_SCANDANAVIAN = "Scandanavian";
+    ETHNICITY_SCOTTISH = "scoțian";
+    ETHNICITY_SERB = "Sârb";
+    ETHNICITY_SINHALESE = "Sinhaleză";
+    ETHNICITY_SLOVAK = "slovac";
+    ETHNICITY_SLOVENE = "sloven";
```
Comment (gitrous) [inline Localization/lang-ro.lua]: Should be capitalized

File: Localization/lang-ro.lua [hunk 2]
```diff
@@ -94,10 +94,136 @@ module.GetDate = function(day,month,year)
...
+    ETHNICITY_PUNJABI = "Punjabi";
+    ETHNICITY_QUECHUA = "Quechua";
+    ETHNICITY_ROMANIAN = "român";
+    ETHNICITY_RUSSIAN = "Rus";
+    ETHNICITY_SAMOAN = "Samoan";
+    ETHNICITY_SCANDANAVIAN = "Scandanavian";
+    ETHNICITY_SCOTTISH = "scoțian";
+    ETHNICITY_SERB = "Sârb";
+    ETHNICITY_SINHALESE = "Sinhaleză";
+    ETHNICITY_SLOVAK = "slovac";
```
Comment (gitrous) [inline Localization/lang-ro.lua]: Should be capitalized

File: Localization/lang-ro.lua [hunk 3]
```diff
@@ -94,10 +94,136 @@ module.GetDate = function(day,month,year)
...
+    ETHNICITY_POLISH = "polonez";
+    ETHNICITY_POLYNESIAN = "Polinezia";
+    ETHNICITY_PORTUGUESE = "Portughez";
+    ETHNICITY_PUNJABI = "Punjabi";
+    ETHNICITY_QUECHUA = "Quechua";
+    ETHNICITY_ROMANIAN = "român";
+    ETHNICITY_RUSSIAN = "Rus";
+    ETHNICITY_SAMOAN = "Samoan";
+    ETHNICITY_SCANDANAVIAN = "Scandanavian";
+    ETHNICITY_SCOTTISH = "scoțian";
```
Comment (gitrous) [inline Localization/lang-ro.lua]: Should be capitalized

File: Localization/lang-ro.lua [hunk 4]
```diff
@@ -94,10 +94,136 @@ module.GetDate = function(day,month,year)
...
+    ETHNICITY_PALAUAN = "Palauan";
+    ETHNICITY_PASHTUN = "Pashtun";
+    ETHNICITY_PERSIAN = "Persan";
+    ETHNICITY_PEUHL = "Peuhl";
+    ETHNICITY_POLISH = "polonez";
+    ETHNICITY_POLYNESIAN = "Polinezia";
+    ETHNICITY_PORTUGUESE = "Portughez";
+    ETHNICITY_PUNJABI = "Punjabi";
+    ETHNICITY_QUECHUA = "Quechua";
+    ETHNICITY_ROMANIAN = "român";
```
Comment (gitrous) [inline Localization/lang-ro.lua]: Should be capitalized

File: Localization/lang-ro.lua [hunk 5]
```diff
@@ -94,10 +94,136 @@ module.GetDate = function(day,month,year)
...
+    ETHNICITY_CZECH = "Ceh";
+    ETHNICITY_DUTCH = "Olandez";
+    ETHNICITY_EGYPTIAN = "Egiptean";
+    ETHNICITY_ESTONIAN = "eston";
+    ETHNICITY_EUROPEAN = "European";
+ETHNICITY_FIJIAN = "Fijian";
+    ETHNICITY_FINN = "Finn"; - Sau finlandez
+    ETHNICITY_FLEMING = "Fleming";
+    ETHNICITY_FON = "Fon";
+    ETHNICITY_FRENCH = "francez";
```
Comment (gitrous) [inline Localization/lang-ro.lua]: Should be capitalized

File: Localization/lang-ro.lua [hunk 6]
```diff
@@ -94,10 +94,136 @@ module.GetDate = function(day,month,year)
...
+    ETHNICITY_JEWISH = "Evreu";
+    ETHNICITY_KAZAK = "Kazak";
+    ETHNICITY_KOREAN = "coreean";
+    ETHNICITY_KURD = "Kurd";
+    ETHNICITY_KUWAITI = "Kuwaiti";
+    ETHNICITY_KYRGYZ = "Kârgâz";
+    ETHNICITY_LAOLOUM = "Lao Loum";
+    ETHNICITY_LATVIAN = "leton";
+    ETHNICITY_LITHUANIAN = "lituanian";
+    ETHNICITY_MACEDONIAN = "macedonean";
```
Comment (gitrous) [inline Localization/lang-ro.lua]: Should be capitalized.

File: Localization/lang-ro.lua [hunk 7]
```diff
@@ -94,10 +94,136 @@ module.GetDate = function(day,month,year)
...
+    ETHNICITY_MALAY = "Malay";
+    ETHNICITY_MALTESE = "maltez";
```
Comment (gitrous) [inline Localization/lang-ro.lua]: Should be capitalized.

File: Localization/lang-ro.lua [hunk 8]
```diff
@@ -94,10 +94,136 @@ module.GetDate = function(day,month,year)
...
+    ETHNICITY_MANDE = "Mande";
+    ETHNICITY_MANDINKA = "Mandinka";
+    ETHNICITY_MAUR = "Maur";
+    ETHNICITY_MAURITIAN = "Mauritian";
+    ETHNICITY_MELANESIAN = "Melanesian";
+    ETHNICITY_MESTIZO = "Mestizo";
+    ETHNICITY_MICRONESIAN = "Micronesian";
+    ETHNICITY_MOLDAVIAN = "moldovean";
```
Comment (gitrous) [inline Localization/lang-ro.lua]: Should be capitalized.

File: Localization/lang-ro.lua [hunk 9]
```diff
@@ -94,10 +94,136 @@ module.GetDate = function(day,month,year)
...
+    ETHNICITY_MONGOL = "Mongol";
+    ETHNICITY_MONTENEGRIN = "Muntenegru";
+    ETHNICITY_MOSSI = "Mossi";
+    ETHNICITY_NAURUAN = "Nauruan";
+    ETHNICITY_NORSE = ”Oameni din Nord"; Note: We do not have a specific term for ppl that used to talk Norse
+    ETHNICITY_NORWEGIAN = "Norvegian";
+    ETHNICITY_OROMO = "Oromo";
+ETHNICITY_OVIMBUNDU = "Ovimbundu";
+    ETHNICITY_PAKISTANI = "pakistanez";
```
Comment (gitrous) [inline Localization/lang-ro.lua]: Should be capitalized.

File: Localization/lang-ro.lua [hunk 10]
```diff
@@ -94,10 +94,136 @@ module.GetDate = function(day,month,year)
 	--Convert the day number to a letter from A-L to access the translation
 	local dayLetter = string.char(month + 40);
 	return day .. " " .. module.Translations["MONTH_" .. dayLetter] .. " " .. year;
+ETHNICITY_AFRICAN = "African";
```
Comment (gitrous) [inline Localization/lang-ro.lua]: All of the translations should be in module.Translations

**Comment by GenesisOfHorace** [comment]
--Character Customization errors	CUSTOMIZATION_ERROR_DATAMISSING = "Lipsesc datele necesare.";	CUSTOMIZATION_ERROR_DAYINVALID = "Ziua selectata e invalida.";	CUSTOMIZATION_ERROR_ETHNICITYINVALID = "Etnia selectata e invalida";	CUSTOMIZATION_ERROR_FACEINVALID = "Fata selectata e invalida.";	CUSTOMIZATION_ERROR_FACIALHAIRINVALID = "Parul Facial selectat e invalid.";	CUSTOMIZATION_ERROR_FIRSTNAMEINVALID = "Primul nume selectat este invalid
.";	CUSTOMIZATION_ERROR_GENDERINVALID ="Genul selectat este invalid.";	CUSTOMIZATION_ERROR_GENDERMATCHFAIL = "Primul nume selectat si genul nu se potrivesc.";	CUSTOMIZATION_ERROR_HAIRINVALID = "Parul selectat e invalid.";	CUSTOMIZATION_ERROR_LASTNAMEINVALID = "Ultimul nume selectat este invalid.";	CUSTOMIZATION_ERROR_MONTHINVALID = "Luna selectata este invalida.";	CUSTOMIZATION_ERROR_NOTCUSTOMIZING "Server-ul nu iti recunoaste sesiunea.Va rugam sa reintrati in joc.";	CUSTOMIZATION_ERROR_NOTLEAPYEAR = "29 Februarie poate fi selectata doar intr-un an bisect.";	CUSTOMIZATION_ERROR_REQUIREDARGUMENT = "Lipseste argumentul necesar.";	CUSTOMIZATION_ERROR_SKINCOLORINVALID = "Culoarea pielii selectate este invalida.";	CUSTOMIZATION_ERROR_UNKNOWNARGUMENT = "Argument necunoscut.";	CUSTOMIZATION_ERROR_YEARINVALID ="Anul selectat este invalid.";

**Comment by Generalyuri342** [comment]
GITROG FUCK SAKE HE FORGOT A LOT OF PUNCTUATION

**Pull request closed without merging**